lib/stripe/stripe_client.rb

```diff
@@ -296,6 +296,10 @@ def execute_request(method, path,
     # The original error message is also appended onto the final exception for
     # full transparency.
     NETWORK_ERROR_MESSAGES_MAP = {
+      EOFError => ERROR_MESSAGE_CONNECTION,
+      Errno::ECONNRESET => ERROR_MESSAGE_CONNECTION,
+      Errno::ETIMEDOUT => ERROR_MESSAGE_TIMEOUT_CONNECT,
+      Errno::EHOSTUNREACH => ERROR_MESSAGE_CONNECTION,
```

Comment:
Would you mind re-arranging these so that the map stays alphabetically ordered (within each "section" anyway)?